### PR TITLE
Lukasp/trim testrunner

### PR DESCRIPTION
**Pull request by LukasPukenis**

### CODE REVIEW

File: test/runner.py [hunk 1]
```diff
@@ -27,15 +27,6 @@ def prepare_docker() -> docker.DockerClient:
     client = docker.DockerClient(base_url="unix://var/run/docker.sock")
 
     # Network creation
-    ipv4_net = client.networks.create(
```
Comment (matszczygiel) [inline test/runner.py:30]: Why it's unnecessary?

Comment (LukasPukenis) [reply test/runner.py:30]: there's `net6` just few lines below, so this wasn't necessary

Comment (matszczygiel) [reply test/runner.py:30]: But it doesn't contain an IPv4 range. How is it possible that it still works?

Comment (LukasPukenis) [reply test/runner.py:30]: Basically because net6 contains both :)

**Review by matszczygiel — APPROVED** [review]

**Pull request merged**